internal/v1/handler.go

```diff
@@ -11,6 +11,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/osbuild/image-builder/internal/distribution"
```

Comment:
This import is miss-ordered, it should be in the osbuild local imorts group, but it's a detail :)